### PR TITLE
Added no animation 

### PR DESCRIPTION
**Pull request by nadeermalangadan**
Added no animation if animationDuration is 0

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,6 +23,11 @@ class Modal extends Component {
       if (props.open && props.children !== this.state.children) {
          this.setState({children: props.children});
       }
+      if(props.animationDuration===0){
+        this.state.scale.setValue(1);
+      }else{
+        this.state.scale.setValue(this.props.open? 1 : 0.8);
+      }
```
Comment (httnn) [inline index.js:30]: let's keep the style coherent and nice:
* space after `if` and before `(`
* spaces on both sides of `===`
* space before `{` and after `}`
* space on both sides of `?` and `:` in a ternary expression

sorry that there are no contribution guidelines, I should probably add some! :)

File: index.js [hunk 2]
```diff
@@ -49,21 +54,25 @@ class Modal extends Component {
...
-         Animated.timing(
-            this.state.opacity,
-            {
-               toValue,
-               duration: animationDuration
-            }
-         ).start();
+         if(animationDuration===0){
+           this.state.opacity.setValue(toValue);
+         }else {
```
Comment (httnn) [inline index.js]: same here!

**Review by httnn — CHANGES_REQUESTED** [review]

**Review by nadeermalangadan — COMMENTED** [review]
code formatted

**Comment by httnn** [comment]
almost there, please still always use `{` and `}` with if statements even if they are only one line!

**Comment by nadeermalangadan** [comment]
update Pull request

**Review by httnn — APPROVED** [review]

**Pull request merged**